backend/pkg/helm/handler.go

```diff
@@ -39,7 +39,8 @@ func NewActionConfig(clientConfig clientcmd.ClientConfig, namespace string) (*ac
 		namespace:    namespace,
 	}
 	logger := func(format string, a ...interface{}) {
-		log.Info().Str("namespace", namespace).Msg(format + "\n" + fmt.Sprintf("%v", a))
+		utils.Log(utils.LevelInfo, map[string]string{"namespace": namespace},
+			nil, format+"\n"+fmt.Sprintf("%v", a))
```

Comment:
Should this Sprintf still be there? I think a is passed into the logger already.

Reply:
I think yes, since the last part of Log function is for message so `format+"\n"+fmt.Sprintf("%v", a)` should be kept as it is.